astroquery/vizier/core.py

```diff
@@ -512,19 +514,21 @@ def _args_to_payload(self, *args, **kwargs):
         else:
             columns = self.columns + columns
 
-        # keyword names that can mean 'all' need to be treated separately
-        alls = ['all', '*']
+        # special keywords need to be treated separately
+        # keyword names that can mean 'all'
+        alls = ['all', '**']
         if any(x in columns for x in alls):
             for x in alls:
                 if x in columns:
                     columns.remove(x)
             body['-out.all'] = 2
+        # keyword name that means default columns
+        if '*' in columns:
+            columns.remove('*')
+            columns_default = True
+        else:
+            columns_default = False
 
-        # process: columns - always request computed positions in degrees
```

Comment:
e.g. the Kang example:
```
In [26]: Vizier.get_catalogs(catalog_list.keys(), get_query_payload=False)[1]
Out[26]: 
<Table masked=True length=50>
 BGPS   GLON    GLAT    S40    _13CO    Comp   Amp    VPeak    FWHM    RMS       SimbadName     _RA.icrs _DE.icrs
        deg     deg      Jy   K km / s          K     km / s  km / s    K                         deg      deg   
int16 float32 float32 float32 float32  uint8 float32 float32 float32 float32      bytes18       float64  float64 
----- ------- ------- ------- -------- ----- ------- ------- ------- ------- ------------------ -------- --------
 6275  48.751  -0.142   0.125     11.2    --    3.43    66.6     2.9    0.15 BGPS G48.751-0.142 290.3502  13.9739
 6276  48.775  -0.222   0.107     20.1    --    3.81    46.9     2.6    0.27 BGPS G48.774-0.222 290.4345  13.9575
 6279  48.834   0.154   0.279     24.3     1    1.67    44.9     2.9    0.10 BGPS G48.834+0.154 290.1210  14.1863
 6279  48.834   0.154   0.279     24.3     2    5.27    50.1     2.8      -- BGPS G48.834+0.154 290.1210  14.1863
 6279  48.834   0.154   0.279     24.3     3    1.16    53.2     2.5      -- BGPS G48.834+0.154 290.1210  14.1863
 6280  48.841  -0.482   0.203     23.4     1    4.72    53.2     3.0    0.12 BGPS G48.841-0.482 290.7028  13.8931
 6280  48.841  -0.482   0.203     23.4     2    1.32    57.3     4.0      -- BGPS G48.841-0.482 290.7028  13.8931
```

vs online: 
![screen shot 2016-11-20 at 23 49 56](https://cloud.githubusercontent.com/assets/6788290/20467266/37f4e930-af7c-11e6-89b1-94ce581de13f.png)